src/SonarQube.TeamBuild.Integration/SonarQube.TeamBuild.Integration.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
```

Comment:
Why are we building two versions of this assembly? I thought all of the supporting libraries were target NetStandard.

Also, the zip only includes the NetStandard version of this assembly.

Reply:
Good catch, we don't need 46.